WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1541,8 +1544,10 @@ private void showGutenbergInformativeDialog() {
     private void setGutenbergEnabledIfNeeded() {
         if (AppPrefs.isGutenbergAutoEnabledForTheNewPosts()
```

Comment:
Kinda a nit and I know it's not part of the changeset in this PR but, I find the `isGutenbergAutoEnabledForTheNewPosts` name a bit confusing, especially because of the past tense in `Enabled`. Reading it I thought it would be true if the Gutenberg has been enabled automatically in the past but that's not its implementation. Can we change it to `shouldAutoEnableGutenbergForTheNewPosts`?

Reply:
Renamed with 0cf405d4ec264499095cbf55ccee3a61837e82c8.